rfcs/20190308-addons-proxy-maintainership.md

```diff
@@ -0,0 +1,174 @@
+as well as new functionality, requirements for maintaining and retiring those 
+submodules must be defined and enforced.
+
+This document details requirements and responsibilities for owning a 
+subpackage/submodule that is included in TensorFlow Addons, as well as the 
+periodic review process for all components of Addons.
+
+## Motivation
+
+In this RFC, we are soliciting discussion regarding Addons maintainership 
+and periodic review. This RFC discussion will help the SIG Addons team 
+determine appropriate roles and responsibilities for proxy maintainers.
+
+## Design Proposal 
+
+### Repository Structure
+
+TensorFlow Addons is structured in a into a heirarchy of subpackages 
+and submodules as described in the [python docs](https://docs.python.org/3/tutorial/modules.html#packages).
+
+A newly proposed subpackage will be required to have at least one active proxy 
+maintainer or organization; and the inclusion must be 
+[discussed with the community at large](https://github.com/tensorflow/addons/issues/58) 
+to see if its matches the addons paradigm. A newly proposed submodule or code 
+addition is required to get approval from the governing subpackage maintainers. 
+If a contributor is not willing to maintain a piece of code, 
+subpackage owners will decide if they are willing to maintain 
+the code going forward and if it should be accepted into future addons releases.
+
+TensorFlow Addons would like to build as inclusive of a community as 
+possible. More than one maintainer is encouraged for subpackages and submodules. 
+The Addons core team will send out monthly emails to the addons [Google Group](https://groups.google.com/a/tensorflow.org/forum/#!forum/addons) detailing which subpackages/submodules are light on maintainers and encouraging 
+community support.
+
+### Subpackages 
+
+To date, the `tensorflow/addons` [subpackages](https://github.com/tensorflow/addons#maintainers) include:
+
+| Subpackage    | Maintainers  | Contact Info                        |
+|:----------------------- |:----------- |:----------------------------|
+| tfa.activations | SIG-Addons | addons@tensorflow.org    |
+| tfa.image |  |                                   |
+| tfa.layers | SIG-Addons |     addons@tensorflow.org |
+| tfa.losses | SIG-Addons |     addons@tensorflow.org |
+| tfa.optimizers | SIG-Addons | addons@tensorflow.org |
+| tfa.seq2seq | Google | [@qlzh727](https://github.com/qlzh727) | 
+| tfa.text |  |  |
+
+
+## Maintainership 
+
+### How to Become a Proxied Maintainer
+
+The easiest way to become a proxy maintainer is to respond to our monthly 
+"request for help" emails. Subpackage maintainership will only be 
+granted after substantial contribution has been made in order to limit the 
+number of users with write perission. Contributions can come in the form of 
+issue closings, bug fixes, new code, or optimizing existing code. Submodule 
```

Comment:
or documentation?

Reply:
+1

Reply:
+1, as well! Thanks for recommending the add. :slightly_smiling_face: